### ISSUE TITLE
installation on MAC OSX

### ISSUE THREAD

**Issue by saheredris** [LibrePlan/libreplan#1962]
Hi,
I'm trying to install Libreplan on mac OSX , upon mvn verify, everything passes well except the :

[INFO] Reactor Summary for LibrePlan 1.6.0:
[INFO]
[INFO] LibrePlan .......................................... SUCCESS [ 0.003 s]
[INFO] LibrePlan Business Module .......................... SUCCESS [ 57.288 s]
[INFO] LibrePlan ZK Components Module ..................... SUCCESS [ 3.989 s]
[INFO] LibrePlan Web Client Module ........................ FAILURE [ 13.954 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:15 min
[INFO] Finished at: 2024-04-04T00:28:21+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project libreplan-webapp: Could not resolve dependencies for project org.libreplan:libreplan-webapp:war:1.6.0: Failed to collect dependencies at com.libreplan.java.zk.components:jasperreportcomponent:jar:1.1: Failed to read artifact descriptor for com.libreplan.java.zk.components:jasperreportcomponent:jar:1.1: The following artifacts could not be resolved: com.libreplan.java.zk.components:jasperreportcomponent:pom:1.1 (absent): Could not transfer artifact com.libreplan.java.zk.components:jasperreportcomponent:pom:1.1 from/to thirdparty (https://nexus.libreplan.dev/repository/thirdparty-libreplan16/): sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR] mvn -rf :libreplan-webapp

**Comment by sanbor** [comment]
I was able to set it up in macOS following the setup instructions

**Comment by kwoot** [comment]
Yes, a lot has changed since the initial opening of this ticket.
